spec/methods_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Sergio', 'X') }
+  describe '#make_move' do
+    it 'return true if user move is correct' do
+      expect(new_game.make_move(player.team, 4)).to eql TRUE
+    end
+    it 'return false if the move is not a number' do
+      expect(new_game.make_move(player.team, 'string')).to eql FALSE
+    end
+    it 'return false if the move is not a number between 1-9' do
+      expect(new_game.make_move(player.team, 12)).to eql FALSE
+    end
+  end
+
+  describe '#check_if_win?' do
+    it 'returns false when there is no winner' do
+      expect(new_game.check_if_win?).to eql FALSE
+    end
+  end
```

Comment:
- [ ] **MANDATORY** Please add a test case for when there IS a winner
![image](https://user-images.githubusercontent.com/53256139/96549590-4b602d80-1275-11eb-91f7-45dd1596f31e.png)

Reply:
> * [x]  **MANDATORY** Please add a test case for when there IS a winner
>   ![image](https://user-images.githubusercontent.com/53256139/96549590-4b602d80-1275-11eb-91f7-45dd1596f31e.png)
- The thing there is that you cannot have a winner and a loser at the same time.
- When there is a winner and when there is no winner are mutually exclusive.